app/src/main/java/org/systers/mentorship/remote/datamanager/UserDataManager.kt

```diff
@@ -43,4 +44,13 @@ class UserDataManager {
     fun updateUser(user: User): Observable<CustomResponse> {
         return apiManager.userService.updateUser(user)
     }
+
+    /**
+     * This will call the getHomeStats method of the UserService
```

Comment:
This can be seen by your code, you can just simply say: `This function fetches user statistics`

Reply:
Okay, I've changed this.